recipes/nsls2-analysis/meta.yaml

```diff
@@ -0,0 +1,217 @@
+    - memory_profiler
+    - mendeleev
+    - mkl ={{ mkl_pin }}
+    - modestimage
+    - napari >=0.3.7
+    - natsort
+    - netcdf4
+    - nexpy >=0.12.7
+    - nodejs
+    - nsls2-detector-handlers >=0.0.2
+    - nslsii >=0.1.1
+    - numexpr
+    - numpy >=1.14.0
+    - oct2py
+    - openpyxl
+    - ophyd >=1.6.0
+    - oscars
+    - papermill
+    - pdfstream
+    - peakutils
+    - periodictable
+    - photutils
+    - pillow
+    - py4xs
+    - pycentroids  # [not win]
+    - pyepics >=3.4.2
+    - pyfai
+    - pyfftw
+    - pymongo >=3.7
+    - pypdf2
+    - pyqt >=5.9.0
+    - pyqtgraph
+    - pystackreg
+    - python
+    - python-graphviz
+    - pyxrf >=1.0.5
+    - pyzbar  # [not win]
+    - qt >=5.9.0
+    - redis-py
+    - reportlab
+    - requests
+    - sasview
+    - scikit-beam >=0.0.23
+    - scikit-learn
+    - scipy
+    - sixtools
+    - slackclient
+    - srwpy
+    - suitcase-csv
+    - suitcase-json-metadata
+    - suitcase-jsonl
+    - suitcase-mongo >=0.3.0
+    - suitcase-msgpack
+    - suitcase-specfile
+    - suitcase-tiff
+    - suitcase-utils
+    - sympy
+    - toml
+    - tomopy
+    - tornado
+    - tqdm
+    - tzlocal >=1.5
+    - xlrd
+    - xlwt
+    - xpdan
+    - xray-vision
+    - xraylarch
+
+test:
+  requires:
+    - nsls2forge-utils
+  imports:
+    - area_detector_handlers
+    - bluesky
+    - dask
+    - databroker
+    - event_model
+    - lmfit
+    - matplotlib
+    - napari
+    - numpy
+    - ophyd
+    - oscars
+    - peakutils
+    - photutils
+    - pymongo
+    - pyxrf
+    - skbeam
+    - srwpy
+    - suitcase.csv
+    - suitcase.json_metadata
+    - suitcase.jsonl
+    - suitcase.mongo_embedded
+    - suitcase.mongo_normalized
+    - suitcase.msgpack
+    - suitcase.specfile
+    - suitcase.tiff_series
+    - suitcase.tiff_stack
+    - suitcase.utils
+    - xray_vision
+    # Beamline-specific packages
+    - chxtools
+    - csxtools
+    - edrixs
+    - eiger_io
+    - hxntools
+    - ispyb
+    - isstools
+    - lixtools
+    - nslsii
+    - py4xs
+    - sixtools
+  files:
+    - test.xlsx
+    - test.xls
+  commands:
+    # Fail if the "conda-forge" channel is in the list of channels.
+    - check-results -t channels -f conda-forge
+    # Fail if versions of the packages are not as expected minimum ones.
+    - check-results -t version -p bluesky     -e 1.6
+    - check-results -t version -p databroker  -e 1.2
+    - check-results -t version -p event_model -e 1.17
+    - check-results -t version -p nslsii      -e 0.1.1
+    - check-results -t version -p ophyd       -e 1.6
+    - python -V
+    - ipython -V
+    - conda-pack --help
+    - napari --help
+    - napari --version
+    - nexpy --help
+    # Test reading the newer format .xlsx with pandas/openpyxl:
+    - python -c "import pandas; print(pandas.read_excel('test.xlsx'))"
+    # Test reading the old format .xls with pandas/xlrd:
+    - python -c "import pandas; print(pandas.read_excel('test.xls'))"
```

Comment:
The `test.xlsx` and `test.xls` files are missing. Please add them to the `recipe` dirs from the nsls2forge feedstocks.